openmdao/core/driver.py

```diff
@@ -48,6 +48,8 @@ def __init__(self):
         self.root = None
 
         self.iter_count = 0
+        self.dv_conversions = {}
```

Comment:
I guess I meant then to declare them in _setup, so you start with an empty dict instead of potentially a dict with stuff in it.  For example, if they set a scaler and setup and run, then set that scaler back to 1.0 and re-setup and re-run. the value in the dict will still have the old scaler value in it.